.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
+node_js:
+  - "5.2"
```

Comment:
likely let this be 5, so it drifts with stable.

cli also supports the commonly used versions:

10
12
4
5